src/dune_rules/context.ml

```diff
@@ -484,8 +483,8 @@ let create ~(kind : Kind.t) ~path ~env ~env_nodes ~name ~merlin ~targets
       Memo.fork_and_join default_library_search_path (fun () ->
           let+ lines =
             Memo.of_reproducible_fiber
-              (Process.run_capture_lines ~display:!Clflags.display ~env Strict
-                 ocamlc [ "-config" ])
+              (Process.run_capture_lines ~display:Quiet ~env Strict ocamlc
+                 [ "-config" ])
```

Comment:
Here we query ocamlc's config. This is internal and does not need to be reported to the user.